src/ca/caChannel.cpp

```diff
@@ -793,14 +797,18 @@ void CAChannelGet::channelCreated(const Status& status,Channel::shared_pointer c
     if(DEBUG_LEVEL>0) {
         std::cout << "CAChannelGet::channelCreated " <<  channel->getChannelName() << endl;
     }
+    firstTime = true;
     ChannelGetRequester::shared_pointer getRequester(channelGetRequester.lock());
     if(!getRequester) return;
     chtype newType = getDBRType(pvRequest, channel->getNativeType());
     if(newType!=getType) {
         getType = getDBRType(pvRequest, channel->getNativeType());
         pvStructure = createPVStructure(channel, getType, pvRequest);
         bitSet = BitSetPtr(new BitSet(pvStructure->getStructure()->getNumberFields()));
-        bitSet->set(0);
+        pvCopy = PVCopy::create(
+          createPVStructure(channel, getType, pvRequest),
+          CreateRequest::create()->createRequest("field()"),
```

Comment:
Is there more work to be done here?  Why is the provided ```pvRequest``` variable not passed to ```PVCopy::create()```?  I would naively assume that it should be.

Reply:
The create code is:
PVCopyPtr PVCopy::create(
    PVStructurePtr const &pvMaster, 
    PVStructurePtr const &pvRequest, 
    string const & structureName)
{
    PVStructurePtr pvStructure(pvRequest);
    if(structureName.size()>0) {
        if(pvRequest->getStructure()->getNumberFields()>0) {
            pvStructure = pvRequest->getSubField<PVStructure>(structureName);
            if(!pvStructure) return PVCopyPtr();
        }
    } else if(pvStructure->getSubField<PVStructure>("field")) {
        pvStructure = pvRequest->getSubField<PVStructure>("field");
    }
    PVCopyPtr pvCopy(new PVCopy(pvMaster));
    bool result = pvCopy->init(pvStructure);
    if(!result) pvCopy.reset();
    return pvCopy;
}

The pvRequest structure is passed to pvCopy->init